src/Input/style.css

```diff
@@ -48,12 +48,22 @@
 
     .ant-input-search .ant-input-group .ant-input-affix-wrapper:not(:last-child) {
       @apply rounded-l-full;
-      @apply border-r-transparent z-0;
+      @apply border-transparent z-0;
     }
 
     .ant-input-search > .ant-input-group > .ant-input-group-addon:last-child {
       border: 1px solid;
-      @apply rounded-r-full bg-white border-l-0 border-neutral-300;
+      @apply rounded-r-full bg-white border-l-0 border-transparent;
+    }
+
+    .ant-input-search {
+      .ant-input-affix-wrapper-focused {
+        @apply border-neutral-300 shadow-none !important;
+      }
+
+      .ant-input-affix-wrapper-focused + .ant-input-group-addon:last-child {
+        @apply border-neutral-300 shadow-none !important;
```

Comment:
why the need to use important here?

Reply:
because of the input component is using -focused classes to control the border color. if didn't put important here, it will not overwrite the style.